robotbase/src/main/java/frc/robot/subsystems/LEDs.java

```diff
@@ -0,0 +1,50 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.AddressableLED;
+import edu.wpi.first.wpilibj.AddressableLEDBuffer;
+import edu.wpi.first.wpilibj.util.Color;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class LEDs extends SubsystemBase {
+  private AddressableLED m_led;
+  private AddressableLEDBuffer m_ledBuffer;
+
+  public boolean ledsOn = false;
+
+  public LEDs() {
+    m_led = new AddressableLED(0);
+    m_ledBuffer = new AddressableLEDBuffer(60);
```

Comment:
No magic numbers, this should be a constant. What happens when we have a strip of 59 or 61 leds?